src/pubsub/handlers/CallMethodHandler.js

```diff
@@ -47,6 +47,15 @@ export default class CallMethodHandler extends BaseHandler {
       }
 
       process.env.TimeoutInMS = message.responseTimeout ? message.responseTimeout : null;
+      let communicationMode = message.context.communicationMode;
+      // TODO: Need to correct after deciding the variable name for SDK 2.0
+      if (process.env.FIREBOLT_V2 >= '2.0.0') {
+        if (process.env.LIFECYCLE_VALIDATION == true) {
+          communicationMode = CONSTANTS.TRANSPORT;
+        }
+      }
+      process.env.COMMUNICATION_MODE = communicationMode;
```

Comment:
Why are we overriding a global communication mode variable here? We want to only have Transport for this one "CallMethod" (And only others that come in for Lifecycle).

Shouldn't we instead override "message.context.communicationMode"?